index.html

```diff
@@ -10,36 +10,50 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
+<div class="container"> <img src="img/lambda-black.png" alt="toplogo">
+<header>
+<nav>
+    <a href="#">Home</a>
+    <a href="about.html">About</a>
+    <a href="#">Products</a>
+    <a href="#">Blog</a>
+    <a href="#">Contact</a>
+</nav>
+</header>
+</div>
 
+<section class="top-content">
+<div class="jpg">
+<img src="img/jumbo.jpg">
+</div>
 <body>
     <div class="container">
    
-        <section class="top-content">
-            <div class="text-container">
-                <h2>The Future</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+        <div class="top-content">
+            <header><h2>The Future</h2></header>
+                <p>Proin sed quam sed tellus vestibulum ultrices quisin nunc. Phasellus id dui id tortor tincidunt efficitur.Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
-            <div class="text-container">
-                <h2>The Past</h2>
+            <div class="top-content">
+                <header><h2>The Past</h2></header>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
-        </section>
+        </div>
         
-        <section class="middle-content">
+        <div class="middle-content">
         
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box1">Box 1</div>
+                <div class="box2">Box 2</div>
```

Comment:
You almost had it, the boxes in your live view are altered because you removed the styles for each box. In this case, you could've kept the "box" class and added an additional class to each box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```